SHIELD.md

```diff
@@ -0,0 +1,68 @@
+#Activity Stream SHIELD Study
+
+## How to run this SHIELD study
+
+tldr; Run the command ```npm run once``` and get a randomly assigned variant.
+
+The SHIELD study work currently lives on branch ```shield_study``` in this repo. To run the study, checkout the branch, run command ```npm start``` in one terminal window, and in another terminal window run command ```shield run . -- -b nightly```. This will randomly pick one of the two variants provided in the study and start it in Nightly. At this point you can see in the terminal which variant it chose (it'll read either 'load: ActivityStream' or 'load: Tiles'). You can click around and see the ping payloads that get sent. If you want to see one variant specifically, run either ```shield run . Tiles -- -b nightly``` or ```shield run . ActivityStream -- -b nightly``` instead to load up the variant specified.
+
+## How it works
+
+#### Specifics of this study
+In this [SHIELD study](https://wiki.mozilla.org/Firefox/Shield/Shield_Studies) a user will be randomly assigned into either the control group or the variant group. If they are in the control group, Activity Stream will **not** be loaded into their browser, and instead they will have the original about:newtab page as their default.
+#### About the metrics
+At this point any interaction that the user makes with about:newtab will send a ping to our custom data pipeline where it will be recorded for analysis. If they are in the variant group, normal Activity Stream will be loaded and all user events will be collected and recorded as normal. The pings belonging to a SHIELD study user will contain an extra field named ```shield_variant``` which will contain either the name of the control group or the name of the variant group. This will be what we use to distinguish which group each unique user belongs to.
+
+**Sample ping:**
+```json
+{
+	"event":"CLICK",
+	"action_position":0,
+	"source":"AFFILIATE",
+	"shield_variant":"shield-study-01-Tiles",
+	"action":"activity_stream_event",
+	"tab_id":"-3-2",
+	"client_id":"ee88c4ef-4f1d-8149-9582-b727dc92e89f",
+	"addon_version":"1.1.5",
+	"locale":"en-US",
+	"page":"NEW_TAB",
+	"session_id":"{ab6f3b3e-dfc6-f944-b082-6f4bbbc21e72}"
+}
+```
```

Comment:
Would be good to include the search ping (to be clear that we aren't sending the query) and the session ping (shows the history and bookmark size, etc).